pkg/cluster/node.go

```diff
@@ -65,9 +78,9 @@ func createNode(name, image, clusterLabel string) (handle *nodeHandle, err error
 			"--name", name, // ... and set the container name
 			// label the node with the cluster ID
 			"--label", clusterLabel,
-			"--expose", "6443", // expose API server port
-			// pick a random ephemeral port to forward to the API server
-			"--publish-all",
+			// publish selected port for the API server
+			"--expose", fmt.Sprintf("%d", port),
+			"-p", fmt.Sprintf("%d:%d", port, port),
```

Comment:
If we keep this as %d:6443, we can avoid the config template change I think? Not sure what the original motivations for the PR are

Reply:
making sure things consistently hit the correct port by informing everything what the api server port is.

Reply:
i assume that the random port is motivated by security?

Reply:
actually because we support multiple clusters on one machine and we set localhost:port to point to them, we already set a random port today, we just don't use the same port within the container which makes some inconsistency with the api server port